Services/CO.CDP.Organisation.WebApi/UseCase/CreateOrganisationSignMouUseCase.cs

```diff
@@ -38,6 +41,63 @@ public async Task<bool> Execute((Guid organisationId, SignMouRequest signMouRequ
 
         organisationRepository.SaveOrganisationMou(mouSignature);
 
+        await NotifyCopyOfMoURequest(mouSignature, organisation);
+
         return await Task.FromResult(true);
     }
+
+    private async Task NotifyCopyOfMoURequest(Persistence.MouSignature mouSignature, Persistence.Organisation organisation)
+    {
+        var baseAppUrl = configuration.GetValue<string>("OrganisationAppUrl") ?? "";
+        var templateId = configuration.GetValue<string>("GOVUKNotify:MouDataSharingAgreementEmailTemplateId") ?? "";
+
+        var missingConfigs = new List<string>();
+
+        if (string.IsNullOrEmpty(baseAppUrl)) missingConfigs.Add("OrganisationAppUrl");
+        if (string.IsNullOrEmpty(templateId)) missingConfigs.Add("GOVUKNotify:MouDataSharingAgreementEmailTemplateId");
+
+        if (missingConfigs.Count != 0)
+        {
+            logger.LogError(new Exception("Unable to send MoU email"), $"Missing configuration keys: {string.Join(", ", missingConfigs)}. Unable to send MoU email to user.");
+            return;
+        }
+
+        var requestLink = new Uri(new Uri(baseAppUrl), $"/organisation/{organisation.Guid}/review-and-sign-memorandom?handler=download").ToString();
```

Comment:
This link will always fetch the latest copy of the MoU I believe. But if they accessed this email in 1 month's time, and we have published a new MoU in the meantime, we would need this link to download the copy of the MoU at the time that they signed it.